packages/backend/src/controllers/authentication.ts

```diff
@@ -73,6 +75,104 @@ export const getOpenIdUserFromRequest = ({
     return undefined;
 };
 
+export async function setupClient(): Promise<Client> {
```

Comment:
shouldn't need to export if we move the handler from apiV1.ts

Reply:
done!